moveit_ros/moveit_servo/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 3.5)
 project(moveit_servo)
 
 set(SERVO_LIB_NAME moveit_servo)
```

Comment:
Please add a comment for what each of these is. Remember that to most people, it's hard to tell the difference between `moveit_servo` and `servo_server`. I think that should be:

```
# Core Servo library, for calculations and collision checking. Provides a C++ interface
set(SERVO_LIB_NAME moveit_servo)
# Library for parameter loading
set(SERVO_PARAM_LIB_NAME ${SERVO_LIB_NAME}_parameters)
# A library to run as a ROS node component, and receive commands via topics
set(COMPOSABLE_NODE_NAME servo_server)
```

Reply:
What do you think of calling the node just `servo` or `servo_node`?  The word `server` doesn't mean much in ros and seems confusing to me.

Reply:
Or maybe even servo_component?

Reply:
i like `servo_component`  :+1: